README.md

```diff
@@ -40,14 +40,26 @@ AZNFS is supported on following Linux distros:
 	```
 	It will install the aznfs mount helper program and the aznfswatchdog service.
 
+## Auto Update AZNFS
+
+- Upon running the installation command, you will be prompted to configure automatic updates for AZNFS. Enabling automatic updates ensures that you 
+  stay current with the latest features, improvements, and security patches, providing you with the best and most seamless AZNFS experience. 
+  You can turn off/on auto-update at any time from /opt/microsoft/aznfs/data/config and update respectively.
```

Comment:
You can also turn off/on auto-update at any time by changing the value of AUTO_UPDATE_AZNFS variable in /opt/microsoft/aznfs/data/config.

There seems to be a repetition of this, so just say once.